BooruSharp/Booru/Template/E621.cs

```diff
@@ -37,33 +38,35 @@ private protected override JToken ParseFirstPostSearchResult(object json)
 
         private protected override Search.Post.SearchResult GetPostSearchResult(JToken elem)
         {
-            // TODO: Check others tags
-            string[] categories =
+            var detailedTags = new List<Search.Tag.SearchResult>();
+            var tags = new List<string>();
+            foreach(var cat in elem["tags"].OfType<JProperty>())
             {
-                "general",
-                "species",
-                "character",
-                "copyright",
-                "artist",
-                "meta",
-            };
+                foreach(var tag in cat.Value.ToObject<string[]>())
+                {
+                    tags.Add(tag);
+                    detailedTags.Add(new Search.Tag.SearchResult(-1, tag, GetTagType(cat.Name), -1));
```

Comment:
For detailed tags maybe we could create a simplified version of SearchResult? Since it looks like first and last param are always -1

Reply:
Sankaku has using post count and id

Reply:
Ah yeah I didn't notice, my bad